CHANGELOG.txt

```diff
@@ -1,13 +1,20 @@
-Airflow 2.2.0b1, TBD
+Airflow 2.2.0b2, TBD
 -------------------------
 
 New Features
 """"""""""""
 
 - AIP-39: Handle DAG scheduling with timetables (#15397)
 - AIP-39: ``DagRun.data_interval_start|end`` (#16352)
+- Add docs for AIP 39: Timetables (#17552)
 - AIP-40: Add Deferrable "Async" Tasks (#15389)
+- Add a Docker Taskflow decorator (#15330)
 - Add Airflow Standalone command (#15826)
+- Production-level support for MSSQL (#18382)
```

Comment:
Probably worth moving to Misc/Internal

```suggestion
- Dockerfile: Production-level support for MSSQL (#18382)
```

Reply:
wdyt? as it is not really Airflow's new feature

Reply:
Probably should be a docs-only change instead on second look.